data-serving/scripts/convert_data.py

```diff
@@ -0,0 +1,55 @@
+# Lint as: python3
+'''Script to convert CSV line-list data into json compliant with the MongoDB schema.'''
+
+import argparse
+import csv
+import logging
+import json
+
+parser = argparse.ArgumentParser(
+    description='Convert CSV line-list data into json compliant with the MongoDB schema.')
+parser.add_argument('--csv_path', required=True)
+parser.add_argument('--json_path', default='cases.json')
+
+
+def main():
+    logging.basicConfig(filename='convert_data.log', level=logging.INFO)
+    args = parser.parse_args()
+    logging.info("Args: %s", args)
+
+    print('Reading data from', args.csv_path)
+    cases = read_csv(args.csv_path)
+
+    print('Converting data to new schema')
+    convert(cases)
+
+    print('Writing results to', args.json_path)
+    write_json(cases, args.json_path)
+
+    print('Great success! 🎉')
+
+
+def read_csv(csv_path):
```

Comment:
If you're loving Typescript, worth considering [typing](https://docs.python.org/3/library/typing.html). Caveat: not widely used, and limited benefits for a one-off script. Just offering.

Reply:
why the heck not!